QUEST.tsv

```diff
@@ -1948,156 +1948,156 @@ QUEST_20150428_001947	$One more thing. Do not consume too much at once, it could
+QUEST_20150428_001974	$All the Cupols here shall defeat demon lords Boolut, Nuaelle, and Baltrus first. {nl}The enemy will panic without their commanders and will be easier to defeat.
+QUEST_20150428_001975	$Cupols who can't attack should concentrate on blocking the crack. {nl}We can't allow more demons from going to the demon lords.
+QUEST_20150428_001976	$How should I live from now on? {nl}I should have moved to Klaipeda.
+QUEST_20150428_001977	$Since you've placed them into the inventory, you can read them anytime.
+QUEST_20150428_001978	$The power of Mirtis was so overwhelming that Cunningham had to make an important decision.
+QUEST_20150428_001979	$I hear the more and more monsters are appearing in the Baron Arleruno's land. {nl}Please defeat those monsters before they take any innocent lives.
+QUEST_20150428_001980	$According to the information I've gathered from the wings of Baibora, the demons come out in groups from a place called Demon's Prison. {nl}Please help us with this to prevent a greater chaos from happening.
+QUEST_20150428_001981	$Help the druids that were dispatched to the farm of the Green Family. {nl}They're looking for people who can help with the mass purification.
+QUEST_20150428_001982	$Dina Bee Farm is in danger of monster attacks. {nl}They can't even work so they desperately need help.
+QUEST_20150511_001983	$It's better to use the cable car than to head straight into the monsters.
+QUEST_20150511_001984	$Just don't believe everything he says. {nl}He was known for being quite the show-off back at home.
+QUEST_20150511_001985	$It would be nice if all these Revelators being here meant that the Goddesses are back.
+QUEST_20150511_001986	$Power is the most important attribute for swordsman. {nl}You could also train yourself to be agile and possess strong stamina. {nl}You should first think about what to become from now on.
+QUEST_20150511_001987	$Power is the most important attribute for archers. {nl}But, of course you can't ignore agile movement. {nl}You should first think about what to become from now on.
+QUEST_20150511_001988	$Intelligence is the most important attribute for Clerics. {nl}But sometimes you many need strong power or spirit. {nl}You should first think about what to become from now on. 
+QUEST_20150511_001989	$Helping reconstruct Fedimian may be more meaningful than pottery. {nl}But people need something to soothe their hearts.
+QUEST_20150511_001990	$If I had the diary, I could continue my adventure, but I wasn't selected… {nl}But if it were you…
+QUEST_20150511_001991	$If you cleverly use the dry air that forms around severe colds, you'd be able to freeze any object even in a desert. {nl}However, if you can't receive the blessing from the Goddess, it won't be possible to make that kind of air.
```

Comment:
The cold (air) is uncountable, so your version is talking about the common cold instead: look [here](http://www.macmillandictionary.com/dictionary/british/cold_2)

Should reword it to "If you cleverly use the dry air that forms around areas of severe cold, you'd be able to..."